test-support/helpers/ember-test-selectors.js

```diff
@@ -5,11 +5,5 @@ const {
 } = Ember;
 
 export default function testSelector(key, value) {
-  let selector;
-  if (!isNone(value)) {
-    selector = `[data-test-${key}="${value}"]`;
-  } else {
-    selector = `[data-test-${key}]`;
-  }
-  return selector;
-};
+  return isNone(value) ? `[data-test-${key}]` : `[data-test-${key}="${value}"]`;
```

Comment:
I like the explicit version better:

```js
if (isNone(value)) {
  return `[data-test-${key}]`;
} else {
  return `[data-test-${key}="${value}"]`;
}
```

But the proposed changes are fine with me too. Definitely better than the `!isNone()`...